.github/workflows/ci.yaml

```diff
@@ -30,8 +34,8 @@ jobs:
         with:
           persist-credentials: false
       - name: 'Setup uv with python ${{ matrix.python-version }}'
-        # v6.8.0
-        uses: 'astral-sh/setup-uv@d0cc045d04ccac9d8b7881df0226f9e82c39688e'
+        # v7.1.2
+        uses: 'astral-sh/setup-uv@85856786d1ce8acfbcc2f13a5f3fbd6b938f9f41'
```

Comment:
per the action distro, seems possible to be less specific than the precise hash: https://github.com/astral-sh/setup-uv?tab=readme-ov-file#install-a-required-version-or-latest-default - there's always going to be some magic to finding a breaking workflow specification, but i've generally found it easy to use a somewhat moving target tag

Reply:
(ibid next entry)